File: find_slides.php
<?php
$lines = file('p.php');
$output = "";
foreach ($lines as $idx => $line) {
    if (strpos($line, 'PREMIUM:') !== false || strpos($line, '_images') !== false || strpos($line, '_video') !== false) {
        if (strpos($line, 'elseif') !== false || strpos($line, 'if') !== false || strpos($line, '$imgs') !== false || strpos($line, '$slide_key') !== false) {
            $output .= "Line " . ($idx + 1) . ": " . trim($line) . "\n";
        }
    }
}
file_put_contents('find_slides_result.txt', $output);
echo "Done! Output written to find_slides_result.txt";

File: find_slides.py
# Search for premium slides in p.php
with open('p.php', 'r', encoding='utf-8') as f:
    lines = f.readlines()

for idx, line in enumerate(lines):
    if 'PREMIUM:' in line or '_images' in line or '_video' in line:
        if 'elseif ($slide_type ===' in line or 'if ($slide_type ===' in line or '$imgs =' in line or '$slide_key =' in line:
            print(f"Line {idx+1}: {line.strip()}")
